lib/marked-renderer.js

```diff
@@ -1,7 +1,21 @@
 const { marked } = require('marked')
 
-/** A custom renderer that avoids wrapping `<img>` tags in `<p>` tags */
+/**
+ * Custom markdown renderer
+ */
 class DesignSystemRenderer extends marked.Renderer {
+  /**
+   * Ensure code blocks can be focused and scrolled
+   * with the keyboard via `tabindex="0"`
+   */
+  code (code, language, isEscaped) {
+    return super.code(code, language, isEscaped)
+      .replace('<code', '<code tabindex="0"')
```

Comment:
Ensure we add `tabindex="0"` to Markdown code blocks

This maintains keyboard focus + arrow key scrolling as added in:

* https://github.com/alphagov/govuk-design-system/pull/1351